edb/server/protocol/binary.pyx

```diff
@@ -1187,6 +1195,9 @@ cdef class EdgeConnection:
                 elif k == QUERY_HEADER_ALLOW_CAPABILITIES:
                     self.version_check("ALLOW_CAPABILITIES header", (0, 9))
                     allow_capabilities = parse_capabilities_header(v)
+                elif k == QUERY_HEADER_EXPLICIT_OBJECTIDS:
+                    self.version_check("EXPLICIT_OBJECTIDS header", (0, 9))
```

Comment:
Perhaps we should bump the proto to 0.10

Reply:
changed